blogposts/2021/dev-tool-time-adam-gordon-bell.md

```diff
@@ -0,0 +1,103 @@
+description: 'Adam Gordon Bell, Developer Advocate at Earthly, shares how he leverages dev tools to write consistent, high-quality technical prose.'
+author: Scott Bailey
+authorUrl: https://about.sourcegraph.com/handbook/company/team#scott-bailey-he-him
+publishDate: 2021-10-06T10:00-07:00
+tags: [blog]
+slug: dev-tool-time-adam-gordon-bell
+heroImage: https://sourcegraphstatic.com/blog/dev-tool-time/adam-gordon-bell/linkedin.jpg
+socialImage: https://sourcegraphstatic.com/blog/dev-tool-time/adam-gordon-bell/linkedin.jpg
+published: true
+---
+
+As a developer advocate, [Adam Gordon Bell](https://twitter.com/adamgordonbell) writes prose as a central part of his work. As a programmer, he wants the same experience writing prose that he has writing code, where linters and build pipelines help with consistency, formatting, and error checking. When Adam joined us for [Dev Tool Time](https://info.sourcegraph.com/dev-tool-time) on September 29, 2021, he shared his favorite dev tools for writing and how Earthly helps keep his published writing high quality.
+
+You can watch the recording of this episode at the [bottom of this blog post](#Watch-the-recording), but first let’s check out Adam’s desk setup:
+
+## Hardware
+
+Adam drives two monitors with a 16" Macbook Pro hooked into a CalDigit Thunderbolt dock. He uses a Logitech gaming mouse, whose thumb buttons ease browser navigation.
+
+![Adam’s current desk setup](https://sourcegraphstatic.com/blog/dev-tool-time/adam-gordon-bell/current-computer.jpeg)
+
+Notably, Adam uses a 45lb iron weight, directly under his Macbook, as a heatsink. Impressively, his sit/stand barely notices the weight, even when it’s joined by Adam’s one-eyed shelter cat, Griswald.
+
+![Adam’s 45lb weight heatsink](https://sourcegraphstatic.com/blog/dev-tool-time/adam-gordon-bell/heatsink.jpeg)
+
+As the host of the popular tech podcast [CoRecursive](https://corecursive.com/), Adam uses a [MixPre-6 audio interface](https://www.sounddevices.com/product/mixpre-6-ii/) and the popular [Electro-Voice RE20 microphone](https://products.electrovoice.com/na/en/re20), and has acoustic paneling in his office.
+
+## The cost of customization
+
+> “If you’re out there and you have a crazy custom keyboard and you’re using the permissive hold feature, just remember that I built that and then gave up on the whole thing.”
+
+Missing from Adam’s current setup is the custom-built and programmed split keyboard he used several years ago. In his focus on configuration, he had even contributed code to the keyboard’s software, code that people are still using today. After his keyboard broke, though, Adam discovered the unpleasantness of no longer being able to comfortably and easily use a standard keyboard, and so switched to using a regular Apple keyboard to avoid the same pain in the future.
+
+![Adam’s 2016 desk setup with custom keyboard](https://sourcegraphstatic.com/blog/dev-tool-time/adam-gordon-bell/2016-computer-ergodox.png)
```

Comment:
I think we could lose this image as well since you can see the setup in the video beneath (it's even visible in the freeze frame for people who don't actually watch the video)